integration-tests/pom.xml

```diff
@@ -82,46 +82,53 @@
 				<artifactId>cds-maven-plugin</artifactId>
 				<version>${cds.services.version}</version>
 				<executions>
-					<!-- Following steps start and stop MTX sidecar for tests executed by failsafe plugin -->
 					<execution>
-						<id>install-sidecar</id>
+						<id>cds.install-node</id>
 						<goals>
-							<goal>npm</goal>
+							<goal>install-node</goal>
 						</goals>
-						<phase>pre-integration-test</phase>
 						<configuration>
-							<arguments>install --no-save</arguments>
-							<skip>${skipTests}</skip>
-							<workingDirectory>${sidecar.dir}</workingDirectory>
+							<skip>${cdsdk-global}</skip>
```

Comment:
I think `${cds.node.executable}` is not set, if the `install-node` goal is skipped. Would be great if this property would default to the executable name `node`, assuming that if `install-node` is not executed it is available in the path.
@mofterdinger Am I correct? What do you think?

Reply:
Yes, you are right, if the goal `install-node` is not executed, the property is not set.
We could try to set the value of the property in the pom.xml first: <cds.node.executable>node</cds.node.executable>.
If the install-node is execute, it will overwrite this value.

Reply:
I'm working on a PR to set just the executable names node/npm/npx in case the goal is skipped.